features/support/world.js

```diff
@@ -100,6 +100,7 @@ function CustomWorld({ attach, parameters }) {
     this.instance = `${this.protocol}://${process.env.SD_API}`;
     this.testOrg = process.env.TEST_ORG;
     this.username = process.env.TEST_USERNAME;
+    this.scmContext = process.env.TEST_SCM_CONTEXT;
```

Comment:
We should add `TEST_SCM_CONTEXT`.
https://github.com/screwdriver-cd/screwdriver/blob/5e014e0c61855edf585ac009f254f08695224140/screwdriver.yaml#L55
https://github.com/screwdriver-cd/screwdriver/blob/5e014e0c61855edf585ac009f254f08695224140/screwdriver.yaml#L83